### PR TITLE
Added tfds dataset example for MLMMaskGenerator

### LINKED ISSUE #120: Add an example to MLMMaskGenerator's docstring to show the best practice
[MLMMaskGenerator](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/layers/preprocessing/mlm_mask_generator.py) is a helpful tool to generate masks for input sequences. The current docstring only has two basic examples, and it would be better for users if we can provide an example of how it should be used in a real MLM workflow. 

Basically the example will have:
1. Prepare data, we can generate random strings or load some small datasets from TFDS (tensorflow datasets).
2. Instantiate a tokenizer, and call tokenize on the data.
3. Apply `MLMMaskGenerator` on the tokenized data.
4. Feed the masked data to a dummy MLM model by calling `model.fit()`.

### PR DESCRIPTION
**Pull request by adhadse**
Resolves #120.
Needs dummy mlm model.

### CODE REVIEW

File: keras_nlp/layers/preprocessing/mlm_mask_generator.py [hunk 1]
```diff
@@ -83,6 +83,45 @@ class MLMMaskGenerator(keras.layers.Layer):
             vocabulary_size=10, mask_selection_rate=0.5, mask_token_id=0, \
             mask_selection_length=5)
     >>> masker(tf.ragged.constant([[1, 2], [1, 2, 3, 4]]))
+
+    With TensorFlow Dataset
```
Comment (mattdangerw) [inline keras_nlp/layers/preprocessing/mlm_mask_generator.py:87]: You can just say "With tf.data"

File: keras_nlp/layers/preprocessing/mlm_mask_generator.py [hunk 2]
```diff
@@ -83,6 +83,45 @@ class MLMMaskGenerator(keras.layers.Layer):
...
+    >>> # Load dataset
```
Comment (mattdangerw) [inline keras_nlp/layers/preprocessing/mlm_mask_generator.py:88]: I'd suggest a fenced code block here (triple backticks), otherwise you need to begin every line with either >>> (new line) or ... (continuation line).

**Review by mattdangerw — CHANGES_REQUESTED** [review]
Some initial comment just to improve the readability here. Thanks!

**Comment by chenmoneygithub** [comment]
@adhadse Hi! I want to check in if there is any update to this? Anything we can help? Thanks!

**Comment by adhadse** [comment]
Hey, I am currently contributing to KerasCV and since my exams are ongoing, I am not working on them either. You can please unassign me.
I'll from now onwards largely contribute there. I hope this doesn't sound rude :)
